web/src/components/Dashboard/RequestPage/RequestPage.tsx

```diff
@@ -0,0 +1,137 @@
+import * as React from "react";
```

Comment:
change this to "import React from "react";" 
react is a very large library. We don't want to import everything as it is extremely inefficient. Only import whatever that is necessary. 
For example, one common import is like "import React, { useState, useEffect } from 'react';"

Reply:
changed into "import React from "react";"